<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>858</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>668</width>
    <height>858</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Chatting</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_connect">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>39</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>连接到服务器</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,0">
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>39</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>39</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_send">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>39</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>39</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
